charts/datadog/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Datadog changelog
 
+## 2.8.4
+
+* Grant access to the `Lease` objects
```

Comment:
For clarity, should we also mention which checks need this?

Reply:
Done!